### ISSUE TITLE
ctrl+click on username nonfunctional

### ISSUE THREAD

**Issue by ghost** [jacob1/The-Powder-Toy#26]
In save browser, ctrl+click on the username of a commenter goes straight back to the simulation window, rather than displaying their profile as before.

When it comes back, the "find and open" button appears to still be active until the mouse is moved.

Then you get a memory read/write error when closing the program.

shift+click and normal click work properly.

**Comment by jacob1** [comment]
Yes, I noticed this. I might disable this functionality until I rewrite the save browser.

**Comment by ghost** [comment]
Even if you don't intend to make the user information appear soon, I think it's important that it not cause a crash when closing.

**Comment by jacob1** [comment]
yeah ... I somewhat forgot about this, although I did get the crash the other day.
